ramalama/console.py

```diff
@@ -0,0 +1,53 @@
+"""
+MIT License
+
+(C) 2024-2025 ramalama developers
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+"""
+import os
+import sys
+import locale
+import logging
+
+def is_locale_utf8():
+    """Check if the system locale is UTF-8."""
+    lang = os.getenv("LC_CTYPE", "") or os.getenv("LANG", "")
```

Comment:
Lets just copy systemd 100%, no need to reinvent the wheel.

If the systemd technique doesn't work for whatever reason, we can open PRs against RamaLama and systemd to fix.

Reply:
So this means just check for "UTF-8" string

Reply:
And we should check LC_CTYPE and LANG separately.

Because of the first or, we won't check both properly.

Reply:
alright